libbeat/outputs/kafka/client.go

```diff
@@ -108,16 +108,28 @@ func (c *client) AsyncPublishEvents(
 			topic = event["type"].(string)
 		}
 
+		var ts time.Time
+		if c.config.Version.IsAtLeast(sarama.V0_10_0_0) {
```

Comment:
A code comment would be nice that quickly explains why whis is only possible with >= 0.10